src/reformatters/common/kubernetes.py

```diff
@@ -8,7 +8,7 @@
 class Job(pydantic.BaseModel):
     model_config = pydantic.ConfigDict(arbitrary_types_allowed=True)
 
-    command: Annotated[Iterable[str], pydantic.Field(min_length=1)]
```

Comment:
Pydantic was coercing lists into some type that can only be iterated once to satisfy this type hint, causing the `command` to get consumed when generating the `job_name`, resulting in an empty `command` below (when used for the K8S command). Adjusting the type hint let the `list` passed in avoid conversion and be iterated more than once.